test/common_utils.py

```diff
@@ -565,14 +552,68 @@ def remove_prefix(text, prefix):
         munged_id = remove_prefix(self.id(), module_id + ".")
         test_file = os.path.realpath(sys.modules[module_id].__file__)
         expected_file = os.path.join(os.path.dirname(test_file),
-                                     "expect",
+                                     "example",
                                      munged_id)
 
         subname_output = ""
         if subname:
             expected_file += "-" + subname
             subname_output = " ({})".format(subname)
-        expected_file += ".expect"
+        expected_file += ".example"
```

Comment:
Are we changing the `.expect` suffix to `.example`? If so, the expect files checked in is not in the correct suffix.

Reply:
I'm not really sure what we should call the files tbh. We could use .expect / .example / something else

Reply:
either .expect or .example works for me, if you are changing this to .example, we also need to update the forward/backward graph file names to sth like`test_lstm_cuda-backward.expect.example`